example-local.yaml

```diff
@@ -3,10 +3,37 @@ packages:
   econet: !include
     # Change this file to the correct one for your device
     file: econet_heatpump_water_heater.yaml
+    # next line is needed for all hvac single zone installs
     # file: econet_hvac.yaml
+    # next line is needed for all hvac multi zone installs (do not include above)
+    # file: econet_hvac_base.yaml
     # Use this if you have a communicating outdoor unit (ODU)
     # files: [econet_hvac.yaml, econet_hvac_odu.yaml]
     # file: econet_tankless_water_heater.yaml
     # file: econet_electric_tank_water_heater.yaml
     vars:
       external_components_source: components
+  
+  # for zones - include the econet hvac package above and the appropriate zones below
+  # because we are re-using esphome-econet for each inclusion, the path is required also.
+  # zone1: !include 
+  #    url: https://github.com/esphome-econet/esphome-econet
+  #   file: econet_hvac_zone_template.yaml
+  #   vars:
+  #     zoneid: "1"
+  #     zonename: "Zone1"
+  #     requestmod: 5
+  # zone2: !include 
+  #    url: https://github.com/esphome-econet/esphome-econet
+  #   file: econet_hvac_zone_template.yaml
+  #   vars:
+  #     zoneid: "2"
+  #     zonename: "Zone2"
+  #     requestmod: 6
+  # zone3: !include 
+  #    url: https://github.com/esphome-econet/esphome-econet
+  #   file: econet_hvac_zone_template.yaml
+  #   vars:
+  #     zoneid: "3"
+  #     zonename: "Zone3"
+  #     requestmod: 7
```

Comment:
This implies users will have to manually configure each zone, correct? Vs. us somehow detecting what zones are present and dynamically creating the right sensors?

Reply:
For single zone installs, user would want to include econet_hvac (line 7) in this file, and that file includes _base.  For multi-zone, econet_hvac_base and all the appropriate zones.  The zone files do not include the _base file due to repeats.

Reply:
Correct.  Dynamically creating the zones would be a far more elegant approach, but not quite sure how to go about that.  Wondering if a reasonable compromise here would be to query if the zone is enabled and surface that to the user in HA?

Reply:
This is refactored with current code.